app/main/views/g_cloud.py

```diff
@@ -340,54 +332,67 @@ def save_search(framework_framework):
         **build_search_query(request.args, filters.values(), content_manifest, lots_by_slug)
     )
 
+    projects = get_direct_award_projects(current_user.id)['open_projects']
+    projects.sort(key=lambda x: x['name'])
+
     return render_template(
         'direct-award/save-search.html',
         framework_framework=framework_framework,
         search_summary_sentence=search_summary.markup(),
         search_api_url=search_api_url,
         form=CreateProjectForm(),
+        projects=projects,
     )
 
 
 @direct_award.route('/<string:framework_framework>/projects/create', methods=['POST'])
 def project_create(framework_framework):
     form = CreateProjectForm()
-    if form.validate_on_submit():
-        name = form.name.data
+    name = form.name.data
+
+    if not form.validate_on_submit() or not request.form['save_search_selection']:
+        projects = get_direct_award_projects(current_user.id)['open_projects']
+        projects.sort(key=lambda x: x['name'])
```

Comment:
There's some duplicate code here; I think I'm planning to roll `project_create` and `save_search` into one view, so don't necessarily worry about doing that before this lands as I can pick that up.